README.md

```diff
@@ -7,51 +7,53 @@ This project was built using only built in web services (ie no NPM packages besi
 ## 🔑 Technical Highlights and Key APIs
 
 ### 💾 System File Access API
-- Save/Save As
-- Open
-- Limitations
-  - Privacy on the web (Losing the file handler on refresh)
-  - Requires user permission
-
-### ✒️ System Font API
-- Read fonts from users OS
-- Fall back on the default list
-- Limitations
-  - Are there stytles in the OS that aren't supported in CSS
-  - Requires user permission
+
+The System File Access API is a MUST inlcude for any sort of app that is going to want to read/write user files. For the Notepad PWA, we used this API to handle opening and saving files. Even though this API requires permission (which we will talk more about later), it is central to the apps functioning, so we will rely on the user accepting this API to have access to their file system in order for the app to work as expected.
+
+### ✒️ Local Font Access API
+
+The Notepad native application offers a lot of different font families and styles to select and use in the editor. Instead of manually porting the fonts over to the PWA, we can just use the Local Font Access API to grab all available fonts on the users operating system. In the spirit of PWA, this will allow the available fonts to  be different across different devices (ie Android will have different font selections compared to Windows).
```

Comment:
```suggestion
The Notepad native application offers a lot of different font families and styles to select and use in the editor. Instead of manually porting the fonts over to the PWA, we can just use the Local Font Access API to grab all available fonts on the users operating system. In the spirit of PWA, this will allow the available fonts to be different across different devices (i.e. Android will have different font selections compared to Windows).
```